pylama/riscv64.patch

```diff
@@ -0,0 +1,27 @@
+diff --git PKGBUILD PKGBUILD
+index 50a2f0c16..19897d19c 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -10,16 +10,19 @@ arch=('any')
+ url="https://github.com/klen/pylama"
+ license=('GPL3')
+ depends=('python-pycodestyle' 'python-pydocstyle' 'python-pyflakes' 'python-mccabe')
+-checkdepends=('python-pytest' 'git' 'mypy')
++checkdepends=('python-pytest' 'git' 'mypy' 'python-tomli')
```

Comment:
https://github.com/klen/pylama/search?q=tomli pylama does not mention tomli anywhere. This should be fixed elsewhere.